distribution/ddf-common/src/main/resources-filtered/etc/org.apache.karaf.features.xml

```diff
@@ -39,7 +39,76 @@
     <overrideBundleDependency/>
 
     <!-- A list of bundle URI replacements that allows changing external feature definitions -->
-    <bundleReplacements/>
+    <bundleReplacements>
+        <bundle originalUri="mvn:org.eclipse.jetty.http2/http2-client/9.4.57.v20241219"
```

Comment:
Also, it would be a good idea to add a comment here explaining why these bundles are being replaced.